services/roomService/src/client/CoveyTownsSocket.test.ts

```diff
@@ -54,119 +60,283 @@ describe('TownServiceApiSocket', () => {
   });
   it('Rejects invalid CoveyTownIDs, even if otherwise valid session token', async () => {
     const town = await createTownForTesting();
-    const joinData = await apiClient.joinTown({coveyTownID: town.coveyTownID, userName: nanoid()});
-    const {socketDisconnected} = TestUtils.createSocketClient(server, joinData.coveySessionToken, nanoid());
+    const joinData = await apiClient.joinTown({
+      coveyTownID: town.coveyTownID,
```

Comment:
i dont know why this looks like i have a bunch of spaces between these lines when i open it in vscode its doesnt look like that